db/migrate/20221221201733_add_index_to_doc_offenses.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToDocOffenses < ActiveRecord::Migration[6.0]
+  def change
+    add_index :doc_offense_codes, [:statute_code, :description, :is_violent ],unique:true, name: 'doc_offense_codes_index'
```

Comment:
This allows us to do upsert for doc_offense_codes too (need the unique index on the combined fields)